internal/pkg/event/event.go

```diff
@@ -52,17 +52,13 @@ type ObserveNotifier interface {
 
 // Embeddable is a type that implements sane defaults as an observer.
 type Embeddable struct {
-	channel Channel
-	types   []Type
+	Chan  Channel
```

Comment:
This is so that if we ever want to make a non-blocking bus, we could.

Reply:
now I'm confused, it looks to be uninitialized in `machined` main.go?